### PR TITLE
Fix icons

### PR DESCRIPTION
**Pull request by herrethan**
This ensures that all our ionicons are properly coded to inherit stroke color or fill color. The essential changes are:
- when `stroke` is defined, remove it and any `strokeOpacity` defined, 
  - need to include `fill="transparent"` and declare `stroke="currentColor"` top level for proper rendering
- when only path is defined, remove any `path` and `pathOpacity` declarations. this ensures it inherits fill from parent

For example: 
image

### CODE REVIEW

**Review by romangorelik — APPROVED** [review]
sweeet!

**Comment by romangorelik** [comment]
I was thinking maybe we should now throw up some page with the new icons and their name so that it is easier to switch icons, either a readme or try to import into graycliff

**Review by ironprice91 — APPROVED** [review]
nice 🤘

**Comment by herrethan** [comment]
@romangorelik yeah totally, the idea was to do that in graycliff. hoping to get to that soon.

File: src/index.tsx [hunk 1]
```diff
@@ -178,160 +19,8 @@ function ReonomyStyles({ children, fontDirectory }: StylesProps) {
...
-  IconShareSocialOutline,
-  IconSpreadsheetOutline,
-  IconSquareOutline,
-  IconTodayOutline,
-  IconTrashOutline,
-  IconWarningOutline
-};
+export { ReonomyPalette, ReonomyCssBaseline, ReonomyTheme };
+
+export * from './ionicons';
```
Comment (adarsh-murthy) [inline src/index.tsx:24]: 👏

**Review by adarsh-murthy — APPROVED** [review]

File: src/palette.tsx [hunk 2]
```diff
@@ -13,14 +11,11 @@ const ReonomyPalette = {
 
   lemon: '#F6EC0E', // yellow
   texasRose: '#FFB74D', // golden orange-yellow
-  blueZodiac: '#0F1144', // dark navy blue
```
Comment (jdents) [inline src/palette.tsx:16]: This is being used in a few places in the app. I just used it myself to match design. Can we get design on the same page with colors? They're not in the material ui for reonomy page but many of these colors are regularly used in figma.

**Review by jdents — APPROVED** [review]
Sweet! Does this mean I can just use `color: 'lime'` and not have to set fill or stroke depending on the svg?

File: src/palette.tsx (hunk 2 above)
Comment (herrethan) [reply src/palette.tsx:16]: this is true. hOweVer... blueZodiac has immediate plans of disappearing
https://reonomy.atlassian.net/browse/SG-23
i figured id get a head start. 
trouble is, we are still using the local /styles/palette.tsx around the app quite a bit

**Comment by herrethan** [comment]
> Sweet! Does this mean I can just use `color: 'lime'` and not have to set fill or stroke depending on the svg?

correct! both line and path SVGs will inherit parent `color` through and through

**Pull request merged**